TestPullRequest/src/testPackage/CalcInterface.java

```diff
@@ -2,7 +2,7 @@
 
 public interface CalcInterface extends CalcInterface2 {
 
-	public void testInterface2();
+	public void newNamePlus();
```

Comment:
@LHommeDeBot Can you please rename this method to something like 'veryNewName'???? Check out 'https://www.example.com/foo/?bar=baz&inga=42&quux' for more information. Also contact me at 'fakemail@fake.com' as soon as you are done.